ethcore/evm/src/interpreter/gasometer.rs

```diff
@@ -234,6 +235,23 @@ impl<Gas: evm::CostType> Gasometer<Gas> {
 
 				Request::GasMemProvide(gas, mem, Some(requested))
 			},
+			instructions::STATICCALL => {				
+				let code_address = u256_to_address(stack.peek(1));
+				let gas = if Builtin::in_precompiles_address_range(&code_address) {
```

Comment:
we're importing ethcore-builtin here just for this function, but it's only used here, which makes me question why was it put in ethcore-builtin in the first place?

Reply:
Tbh, I have been looking at which place fits better this function and I haven't found any that I was just comfortable with. Nonetheless, yes, this is far for perfect if creates more dependencies.

Maybe in `Gasometer::`  is a better place?

Reply:
I think it could just be a standalone function in this module.